### DISCUSSION TITLE
Woefully Behind in Upgrades on this project

### DISCUSSION THREAD

**Discussion by flashfearless** [LuckyPennySoftware/AutoMapper#3954] [NA]
hello all, I recently took over the dev manager role on a call center based app for an insurance company and I see they are using AutoMapper 4.x on a .net 4.5.2 app.  

I realize I need to get this beast to .net core eventually, but was hoping for an interim approach.  I'm trying to cherry pick a version of Automapper I can successfully run in .net 4.8.  The highest explicit support I see it 4.6.1, which really is no better than 4.5.2.

Any help or advice of a direction without refactoring the app to .net core?

**Comment by jbogard** [comment]
Are you trying to upgrade AutoMapper? If so I'd just go one version at a time, there are upgrade guides for each major version (that had major breaking changes).

The .NET 4.6.1 vs. .NET 4.8 support is irrelevant, it runs just fine. That's just the minimum version we support, it also supports everything above that.

**Reply by flashfearless** [reply]
I really have no choice other than to leap from 4.x to 10.x. 

Thanks for clarification on the dependency.

**Reply by jbogard** [reply]
> I really have no choice other than to leap from 4.x to 10.x.

Well GOOD LUCK SIR

**Reply by flashfearless** [reply]
I appreciate the speedy reply and the challenge ahead
